client/audiodevs/PipeWire/pipewire.c

```diff
@@ -162,26 +218,39 @@ static void pipewire_playbackStart(int channels, int sampleRate)
 
   pipewire_playbackStopStream();
 
+  int bufferFrames = sampleRate / 10;
+
   pw.playback.channels   = channels;
   pw.playback.sampleRate = sampleRate;
   pw.playback.stride     = sizeof(uint16_t) * channels;
-  pw.playback.buffer     = ringbuffer_new(sampleRate / 10,
+  pw.playback.buffer     = ringbuffer_new(bufferFrames,
       channels * sizeof(uint16_t));
 
+  int maxLatencyFrames = bufferFrames / 2;
+  char * maxLatency;
+  alloc_sprintf(&maxLatency, "%d/%d", maxLatencyFrames, sampleRate);
```

Comment:
do we really need an alloc here? the string is never going to be more then a handful of bytes, ie `snprintf` on a `char[32]`